applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -38,6 +36,9 @@ static struct bt_audio_lc3_preset lc3_preset = BT_AUDIO_LC3_BROADCAST_PRESET_48_
 static const uint32_t bis_index_mask = BIT_MASK(ARRAY_SIZE(streams) + 1U);
 static uint32_t bis_index_bitfield;
 
+static le_audio_receive_cb receive_cb;
+static bool init_routine_completed;
```

Comment:
Is this a left-over from something else? This static is declared here, but never used?

Reply:
Do you meen init_routine_completed?
This bool is used several places in this file

Reply:
Ah, my bad. Didn't catch the deletion on line:24, so I thought this boolean was created in this PR